src/cudamatrix/cu-device.h

```diff
@@ -26,6 +26,7 @@
 #if HAVE_CUDA == 1
 
 #include <cublas_v2.h>
+#include <cudnn.h>
```

Comment:
Installation is not easy. You need to register as a developer on NVIDIA's website, and you cannot redistribute the binary: "NVIDIA hereby grants Licensee the following limited, non-exclusive, **non-transferable**, non-sublicensable right to use the SOFTWARE"